components/FormModal.js

```diff
@@ -0,0 +1,53 @@
+import styled from "styled-components";
+import { useEffect } from "react";
+import Form from "@/components/Form";
+import SendingConfirmation from "@/components/SendingConfirmation";
+
+export default function FormModal({ showModal }) {
+  useEffect(() => {
+    document.body.style.overflow = "hidden";
+    return () => {
+      document.body.style.overflow = "unset";
+    };
+  }, []);
```

Comment:
why don't you remove the hidden property from the body in the createGlobalStyle of the app?